src/main/java/org/amahi/anywhere/server/model/ServerFile.java

```diff
@@ -43,7 +43,18 @@
 	@SerializedName("mtime")
 	private Date modificationTime;
 
-	public void setParentFile(ServerFile parentFile) {
+       @SerializedName("size")
+       private long size;
+
+       public void setSize(int size) {
```

Comment:
change this to `long` datatype too or you may remove this method as it is not required.